packages/init/src/lib/setupSentry.ts

```diff
@@ -3,22 +3,38 @@ import semver from "semver";
 
 import { SENTRY_EXPRESS_DSN } from "../constants";
 import { checkIsSentryEnabled } from "./checkIsSentryEnabled";
-import { version } from "../../package.json";
+import { name as pkgName, version as pkgVersion } from "../../package.json";
```

Comment:
It might be cleaner to use `import * as pkg` and `pkg.name`/`pkg.version`. I believe Vite/Rollup will convert it to `import { name, version }` automatically.